### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by lucas-leme**
Adicionei algumas correções no módulo de métricas

### CODE REVIEW

**Review by juliapocciotti — APPROVED** [review]

**Comment by luisaheise** [comment]
nossa @lucas-leme vc fede. Fica pedindo minhas reviews nesses pull request de quant

File: turingquant/metrics.py [hunk 1]
```diff
@@ -5,14 +5,15 @@
 import plotly.express as px
 
 
-def sharpe_ratio(returns, risk_free=0):
+def sharpe_ratio(returns, risk_free=0, time_scale=252):
```
Comment (aateg) [inline turingquant/metrics.py:8]: I don't think time_scale is the better name for this variable. Maybe days_num it's a better option.

File: turingquant/metrics.py [hunk 2]
```diff
@@ -21,55 +22,64 @@ def sharpe_ratio(returns, risk_free=0):
...
+
     cov = np.cov(concat)[0][1]
+
     benchmark_vol = np.var(benchmark)
 
     return cov / benchmark_vol
 
 
-def alpha(start_price, end_price, dps):
+def alpha(start_price, end_price, dividends):
```
Comment (aateg) [inline turingquant/metrics.py:55]: This is a nice improvement, variables should be self explanatory

**Review by aateg — APPROVED** [review]
Nice improvements Leme

**Review by gvmossato — APPROVED** [review]

**Review by anor4k — APPROVED** [review]
lgtm

**Comment by lucas-leme** [comment]
[Revisão de Biblioteca do Quant](https://trello.com/c/GbNmcwml/113-revis%C3%A3o-de-biblioteca-do-quant)

**Pull request merged**